README.md

```diff
@@ -136,6 +134,7 @@ If you like this project stars the repo.
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
**[OPTIONAL]**


- Please, let's ensure we remove all the placeholder text in the readme as these texts are guides to help you customize the project using the given template. let's improve on this to make our readme more professional.

Reply:
### MISSING LICENSE [OPTIONAL]


- Your readme is missing the license section as this is important in making your readme more professional and open for others to reuse your code. I'm marking this optional as this wasn't pointed out by the previous reviewer. Please, let's keep this in mind for a subsequent project. 


![Screenshot from 2023-01-06 16-03-04](https://user-images.githubusercontent.com/59397159/211039134-f61c9bae-a5ac-4e85-9a7f-1629ef3414b6.png)

Reply:
### AUTHOR SECTION [REQUIRED]

- As suggested by the previous reviewer, please, kindly update the author section of your readme by adding the author of the project and ensure it follows the pattern as in the microverse readme [template](https://github.com/microverseinc/readme-template).